cpp/include/cuvs/neighbors/cagra.hpp

```diff
@@ -448,6 +463,23 @@ struct index : cuvs::neighbors::index {
  *   auto distances = raft::make_device_matrix<float>(res, n_queries, k);
  *   cagra::search(res, search_params, index, queries, neighbors, distances);
  * @endcode
+ *   In the above example, the index is populated with the dataset and is ready for search. In
```

Comment:
Again, I think we should be careful about the nomenclature here. When I see "populate", I think of a chunk of memory being materialized into something. In reality, I think what we're doing is attaching a view of the dataset to the index. We should be very explicit here allthewhile attempting to be as consistent with prior naming standards as possible (e.g. "attach" vs "add").